redis_test.go

```diff
@@ -45,7 +45,7 @@ func makeFailedStore() subsystems.ComponentConfigurer[subsystems.PersistentDataS
 }
 
 func verifyFailedStoreError(t assert.TestingT, err error) {
-	assert.Contains(t, err.Error(), "no such host")
+	assert.Contains(t, err.Error(), "lookup")
```

Comment:
The error happens to be different on Github vs CircleCI. This is uber fragile, but I couldn't find a way to assert that it's specifically a dns error.